ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -0,0 +1,325 @@
+// this is a temporary file to assist with the migration from (class-based) AdminUser to (functional) AdminUserProfile
+// for RW-7536
+
+import * as React from 'react';
+import {CSSProperties} from 'react';
+import {Link} from 'react-router-dom';
+import * as fp from 'lodash/fp';
+import {Dropdown} from 'primereact/dropdown';
+
+import {formatFreeCreditsUSD, reactStyles} from 'app/utils';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {
+  AccessModule,
+  AccessModuleStatus,
+  AccountPropertyUpdate,
+  InstitutionalRole,
+  Profile,
+  PublicInstitutionDetails,
+  VerifiedInstitutionalAffiliation
+} from 'generated/fetch';
+import {institutionApi, userAdminApi} from 'app/services/swagger-fetch-clients';
+import {ClrIcon} from 'app/components/icons';
+import {FlexColumn, FlexRow} from 'app/components/flex';
+import {getRoleOptions} from 'app/utils/institutions';
+import {TextInputWithLabel} from 'app/components/inputs';
+import {BulletAlignedUnorderedList} from 'app/components/lists';
+import {TooltipTrigger} from 'app/components/popups';
+import {serverConfigStore} from 'app/utils/stores';
+import {accessRenewalModules, computeDisplayDates, getAccessModuleConfig} from 'app/utils/access-utils';
+import {hasRegisteredTierAccess} from 'app/utils/access-tiers';
+
+export const commonStyles = reactStyles({
+  semiBold: {
+    fontWeight: 600
+  },
+  label: {
+    color: colors.primary,
+    fontSize: '14px',
+    fontWeight: 'bold',
+    paddingLeft: '1em',
+  },
+  textInput: {
+    width: '14rem',
+    opacity: '100%',
+    marginLeft: '1em',
+  },
+  textInputContainer: {
+    marginTop: '1rem',
+  },
+  dropdown: {
+    minWidth: '70px',
+    width: '14rem',
+    marginLeft: '1em',
+  },
+  fadeBox: {
+    margin: 'auto',
+    paddingTop: '1rem',
+    width: '96.25%',
+    minWidth: '1232px',
+    color: colors.primary
+  },
+  userAdminArrow: {
+    backgroundColor: colorWithWhiteness(colors.accent, .85),
+    color: colors.accent,
+    borderRadius: '18px',
+    transform: 'rotate(270deg)'
+  },
+});
+
+export const adminGetProfile = async(usernameWithDomain: string): Promise<Profile> => {
+  return userAdminApi().getUserByUsername(usernameWithDomain);
+}
+
+export const getPublicInstitutionDetails = async(): Promise<PublicInstitutionDetails[]> => {
+  const institutionsResponse = await institutionApi().getPublicInstitutionDetails();
+  const institutions = institutionsResponse?.institutions;
+  return fp.sortBy('displayName', institutions);
+}
+
+/**
+ * Present an ordered list of dollar options with the following values:
+ * $300 to $1000, in $100 increments
+ * $1000 to $10,000, in $500 increments
+ * Plus the user's current quota value, if it's not already one of these
+ */
+export const getFreeCreditLimitOptions = (freeTierDollarQuota?: number) => {
+  const START1 = 300;
```

Comment:
why start1 and not just start?

Reply:
my intention was START1+END1 and START2+END2.  I made a small update which I think improves this.